src/test/java/com/adyen/util/CertificateUtilTest.java

```diff
@@ -1,53 +0,0 @@
-/*
```

Comment:
could you please revert this test class as well?

Reply:
reverted back.